Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/non_rigid_mesh_registration.h

```diff
@@ -0,0 +1,758 @@
+// Copyright (c) 2024 GeometryFactory (France).
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org).
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: GPL-3.0-or-later OR LicenseRef-Commercial
+//
+// Author(s)     : Roberto Dyke, Sven Oesau
+//
+#ifndef CGAL_POLYGON_MESH_PROCESSING_REGISTER_MESH_H
+#define CGAL_POLYGON_MESH_PROCESSING_REGISTER_MESH_H
+
+#include <CGAL/license/Polygon_mesh_processing/miscellaneous.h>
+
+#include <CGAL/config.h>
+
+//#ifdef CGAL_EIGEN3_ENABLED
+
+#include <Eigen/Core>
+#include <Eigen/Dense>
+#include <Eigen/SVD>
+#include <Eigen/Sparse>
+
+#include <CGAL/Named_function_parameters.h>
+#include <CGAL/boost/graph/named_params_helper.h>
+#include <CGAL/boost/graph/Face_filtered_graph.h>
+
+#include <CGAL/Polygon_mesh_processing/compute_normal.h>
+
+#include <CGAL/Simple_cartesian.h>
+#include <CGAL/Kd_tree.h>
+#include <CGAL/Splitters.h>
+#include <CGAL/Search_traits_3.h>
+#include <CGAL/Search_traits_adapter.h>
+#include <CGAL/Orthogonal_k_neighbor_search.h>
+
+#include <CGAL/Aff_transformation_3.h>
+
+namespace CGAL {
+namespace Polygon_mesh_processing {
+namespace internal_registration {
```

Comment:
@sloriot should that become a nested namespace instead of `_registration`?

Reply:
there is no guideline about that but it seems cleaner indeed